### ISSUE TITLE
Please tag version 72 release for gnome 42

### ISSUE THREAD

**Issue by highvoltage** [micheleg/dash-to-dock#1692]
Hi, I noticed there's a version 72 tag for ubuntu-dock, can you also please tag a version for the general version?

thanks!

**Comment by MorganAntonsson** [comment]
You can try checking out commit c41fd38. It seems to be working fairly well with Gnome 42.

**Comment by highvoltage** [comment]
> You can try checking out commit [c41fd38](https://github.com/micheleg/dash-to-dock/commit/c41fd380719fec99f80dfe4ddcc39f0cfa6f659a). It seems to be working fairly well with Gnome 42.

If I package c41fd38, I will have to update again when the tag is released, so it would be ideal if it's tagged now since it's working.

**Comment by MorganAntonsson** [comment]
Ah, OK. I thought you just wanted to get it working on your own desktop. I am not in the developer team so I can't help you, sorry.

**Comment by timocapa** [comment]
> You can try checking out commit [c41fd38](https://github.com/micheleg/dash-to-dock/commit/c41fd380719fec99f80dfe4ddcc39f0cfa6f659a). It seems to be working fairly well with Gnome 42.

Why https://github.com/micheleg/dash-to-dock/commit/c41fd380719fec99f80dfe4ddcc39f0cfa6f659a specifically?

**Comment by MorganAntonsson** [comment]
> > You can try checking out commit [c41fd38](https://github.com/micheleg/dash-to-dock/commit/c41fd380719fec99f80dfe4ddcc39f0cfa6f659a). It seems to be working fairly well with Gnome 42.
> 
> Why [c41fd38](https://github.com/micheleg/dash-to-dock/commit/c41fd380719fec99f80dfe4ddcc39f0cfa6f659a) specifically?

Well, I took a look at the git history and found that that commit is the last one on a work branch that was merged to the ubuntu-dock branch on February 23. It seemed like a good candidate so I checked it out, built and installed the extension. I haven't analyzed the code so I don't know if it is ready for release but the extension seems to work so far.

**Comment by skgsergio** [comment]
I installed manually `ubuntu-dock` branch on my Arch Linux with Gnome 42 and so far looks ok (other than the extension is named differently), so I guess I'll keep using it until there is a general release.

**Comment by ugjka** [comment]
> I installed manually `ubuntu-dock` branch on my Arch Linux with Gnome 42 and so far looks ok (other than the extension is named differently), so I guess I'll keep using it until there is a general release.

Yes, this is the way for time being

**Comment by DenisNovac** [comment]
I've made this [container](https://github.com/DenisNovac/dash-to-dock-build-container) if you struggle with building the extension in your workstation like not wanting to install node or something.

**Comment by caduamorimm-dev** [comment]
The incompatibility issue still persists. Did anyone manage to solve it?

**Comment by vijay-prema** [comment]
I installed the `ubuntu-dock` branch and it works but for some reason I cant change the settings, and its not reading the previous settings from my dash-to-dock either. Anyone know whats wrong?

**Comment by vijay-prema** [comment]
> The incompatibility issue in ubuntu 22.04 still persists. Did anyone manage to solve it?

What I did to make it work in Ubuntu 2204 was:
```
git clone git@github.com:micheleg/dash-to-dock.git
git checkout c41fd38
sudo apt install sassc
make -C dash-to-dock install
```
Then restart Gnome by doing `Alt+F2`, type `r`, and press `Enter`.

**Comment by 1player** [comment]
@3v1n0 since you seem to be the one working on the ubuntu-dock branch, why isn't it merged into master yet? What's still missing?

**Comment by 3v1n0** [comment]
Review from another maintainer (@micheleg or @ewlsh).

**Comment by vanvugt** [comment]
fc795c52b7f973eee065547c4d3eb92a6cbada4a was tagged a couple of days ago.

**Comment by hswong3i** [comment]
> [fc795c5](https://github.com/micheleg/dash-to-dock/commit/fc795c52b7f973eee065547c4d3eb92a6cbada4a) was tagged a couple of days ago.

@vanvugt but https://extensions.gnome.org/extension/307/dash-to-dock/ seems still not yet ready for download?

**Comment by vanvugt** [comment]
> > [fc795c5](https://github.com/micheleg/dash-to-dock/commit/fc795c52b7f973eee065547c4d3eb92a6cbada4a) was tagged a couple of days ago.
> 
> @vanvugt but https://extensions.gnome.org/extension/307/dash-to-dock/ seems still not yet ready for download?

Fair enough. I'm sure @micheleg or someone will get to that soon. This issue however was about the absence of a tag, not of a download.

**Comment by cobalt2727** [comment]
When is https://extensions.gnome.org receiving an update?

**Comment by vanvugt** [comment]
Moved to #1750.
